subsystem/cluster_test.go

```diff
@@ -416,6 +416,10 @@ func installCluster(clusterID strfmt.UUID) *models.Cluster {
 	c := reply.GetPayload()
 	Expect(*c.Status).Should(Equal(models.ClusterStatusPreparingForInstallation))
 	generateSuccessfulDiskSpeedResponses(ctx, sdbId, c.Hosts...)
+	for _, h := range c.Hosts {
+		generateContainerImageAvailabilityPostStepReply(ctx, h, []*models.ContainerImageAvailability{common.TestImageStatusesSuccess})
+	}
```

Comment:
Should be under `generateEssentialHostStepsWithInventory` function or a new `generateEssentialPrepareForInstallationSteps`

Reply:
OK